docs/en_US/configuration/configuration.md

```diff
@@ -34,3 +34,15 @@ CONNECTION__EDGEX__REDISMSGBUS__PORT => edgex.redismsgbus.port int etc/connectio
 ```
 
 The environment variables are separated by "__", the content of the first part after the separation matches the file name of the configuration file, and the remaining content matches the different levels of the configuration items. The file name could be `KUIPER` and `MQTT_SOURCE` in the `etc` folder; or  `CONNECTION` in `etc/connection` folder. Otherwise, the file should in `etc/sources` folder.
+
+### command line parameters
+
+eKuiper supports importing configuration from command line parameters, as follows:
+
+| configuration name | type | configuration role |
+|-----------------|--------|---------------------- --------------------------------|
+| loadFileTye | string | Set the way to load files, support "relative" and "absolute" |
+| etc | string | Set absolute path of etc directory |
```

Comment:
Need to explicitly point out the following paths are only supported in absolute mode.